python/dgl/distributed/dist_graph.py

```diff
@@ -271,7 +271,7 @@ def __init__(self, server_id, ip_config, num_clients, graph_name, conf_file):
 
         # Load graph partition data.
         self.client_g, node_feats, edge_feats, self.gpb = load_partition(conf_file, server_id)
-        self.client_g = _copy_graph_to_shared_mem(self.client_g, graph_name)
+        # self.client_g = _copy_graph_to_shared_mem(self.client_g, graph_name)
```

Comment:
This is commented out for test reason. Can we add an option here? @zheng-da

Reply:
if you want to have three sampler server running, you need to set up the "backup" server. @aksnzhy can you provide any suggestions on how to set up backup servers?